amazon/aws-cli/container.yaml

```diff
@@ -0,0 +1,13 @@
+docker: amazon/aws-cli
+url: https://hub.docker.com/r/amazon/aws-cli
+maintainer: 'Amazon Web Services'
```

Comment:
The maintainer value should be the github handle (i.e. username) of the yaml recipe creator